sockeye/encoder.py

```diff
@@ -40,7 +43,14 @@ def get_encoder(config: 'EncoderConfig', prefix: str = '') -> 'Encoder':
     elif isinstance(config, ConvolutionalEncoderConfig):
         return get_convolutional_encoder(config, prefix)
     else:
-        raise ValueError("Unsupported encoder configuration")
+        from .image_captioning.encoder import ImageLoadedCnnEncoderConfig, \
```

Comment:
Why importing inside the if-clause? It also breaks the structure of the full block.

Reply:
We wanted to avoid importing the image captioning code by default in the main package. I think this is fine.